.github/workflows/deploy-release.yml

```diff
@@ -28,3 +28,14 @@ jobs:
         :t-intensifies: `prod` updated to release `${{ github.event.release.tag_name }}`
       failure-message: |
         release `${{ github.event.release.tag_name }}` failed to update `prod`
+
```

Comment:
https://github.com/getsentry/action-release?tab=readme-ov-file#troubleshooting The action suggests doing `actions/checkout` first, it seems like it tries to compute a previous release.